ViennaAdvantage/Areas/VA012/Scripts/model/callouts.js

```diff
@@ -27,6 +27,31 @@
         return "";
 
     };
+
+    //Callout to SetConversionType on statement Line
+    VA012_Contra.prototype.SetConversionType = function (ctx, windowNo, mTab, mField, value, oldValue) {
+        if (this.isCalloutActive()) {
```

Comment:
take this function before callout mark as active

Reply:
Done.